Sources/SwiftProtobuf/Docs.docc/API.md

```diff
@@ -0,0 +1,708 @@
+# API Overview
+
+---
+
+This document explains how you use the code generated by the SwiftProtobuf plugin.
+
+The generated code relies very heavily on the associated SwiftProtobuf library.
+Apart from the actual properties themselves, most of the methods described
+below are not explicit in the generated code.
+Rather, most of them appear in common shared protocols in the runtime library.
+They're collected here to make it easier to understand.
+
+## Message API
+
+Messages in the input proto file generate Swift structs in the result.
+These structs conform to `SwiftProtobuf.Message` and provide Swift properties for every
+field, basic information about the message, standard initializers, and
+serialization and deserialization methods.
+
+Here is a simple proto3 input file to motivate the example below:
```

Comment:
I'd suggest "proto3 syntax" in call cases, since some users seem to start confusing things with version numbers.

Reply:
Good point!  I updated a few places.